### ISSUE TITLE
Import existing volume

### ISSUE THREAD

**Issue by Brulijam** [hardcore-sushi/DroidFS#295]
Hi. Is it possible to import an existing volume into DroidFS?

**Comment by hardcore-sushi** [comment]
Do you mean by clicking on the "+" button, selecting the path to the volume, and then clicking "Add volume"?

**Comment by Brulijam** [comment]
When clicking the + button and selecting the path to an existing volume, I only have a "CREATE VOLUME" button.
I am using DroidFS version 2.1.3, installed via fdroid and want to use it with a gocrypt volume that was created with plasma-vault in KDE.

**Comment by hardcore-sushi** [comment]
OK, what's the retrieved path? Is it the same as what you get from your file explorer? If it is, what's the content of the path?

**Comment by Brulijam** [comment]
Yes they are the same. Path in DroidFS is `/storage/emulated/0/vaults/Stuff.enc` which contains the files gocryptfs.conf, gocryptfs.diriv and srH2S...ANnzA.

**Comment by hardcore-sushi** [comment]
Alright. This may be a permission problem. Is DroidFS able to properly create a volume under `/storage/emulated/0/vaults/`?

If not, ensure it is granted the [MANAGE_EXTERNAL_STORAGE](https://developer.android.com/reference/android/Manifest.permission#MANAGE_EXTERNAL_STORAGE) permission under Android Settings > Apps > Special app access > All files access > DroidFS. However, the app should automatically request it when you click on the folder icon to select the path, doesn't it work for you?

Also, check that the directory permissions are set to:
```
drwxrwx---  root everybody  /storage/emulated/0/vaults/Stuff.enc
-rw-rw----  root everybody  /storage/emulated/0/vaults/Stuff.enc/gocryptfs.conf
-rw-rw----  root everybody  /storage/emulated/0/vaults/Stuff.enc/gocryptfs.diriv
```
